user/dependency_management/gradle-plugin.md

```diff
@@ -43,6 +43,14 @@ gauge {
     additionalFlags = '--verbose'
 }
 
+// copying required dependencies for Gauge (required)
+task copyLibs {
+    copy {
+        from configurations.runtime
+        into "$projectDir/libs"
+    }
+}
+
```

Comment:
Is it possible to add the dependencies to classpath without explicitly copying to `$projectDir/libs`?